lib/seattleflu/api/datastore.py

```diff
@@ -156,6 +156,42 @@ def store_redcap_det(session: DatabaseSession, document: str) -> None:
             raise BadRequestDatabaseError(error) from None
 
 
+@export
+@catch_permission_denied
+def export_metadata_for_augur_build(session: DatabaseSession) -> dict:
```

Comment:
I'd prefer `fetch` over `export`. The rest of the functions in this file are named `store_*` and `fetch` is the clearer complement of `store` to me. (or maybe `get`, but `fetch` aligns with existing functions in other database routines of ours)